Overlay 12

Jiaqi-beep opened the pull request:
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/overlay_12.ncl
- Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/overlay_12_1_lg.png
![image](https://user-images.githubusercontent.com/63561327/125877632-0a93a6c6-11a7-4aef-9fa3-e96ee36c956e.png)

anissa111 reviewed (approved):
This looks perfect!

michaelavs reviewed (approved):
Before merging, I wanted to double check that the changes I made are ok with you. The line breaks I added were mainly for consistency with the rest of the plot. The section I made the changes to (roughly 130 through 180) just didn't have line breaks between each section of the code, but the code above and below this section did. Additionally, I moved the zorder argument it be inside the NaturalEarthFeature call on line 119. If you do not agree with these changes, no worries and I will revert them to how it were before! They were not detrimental but more for formatting purposes than anything else, so I am fine with approving either way!

Jiaqi-beep commented:
> Before merging, I wanted to double check that the changes I made are ok with you. The line breaks I added were mainly for consistency with the rest of the plot. The section I made the changes to (roughly 130 through 180) just didn't have line breaks between each section of the code, but the code above and below this section did. Additionally, I moved the zorder argument it be inside the NaturalEarthFeature call on line 119. If you do not agree with these changes, no worries and I will revert them to how it were before! They were not detrimental but more for formatting purposes than anything else, so I am fine with approving either way!

Hi @michaelavs, I am good with these changes! Thanks for making them.
